### ISSUE TITLE
Open SQLite file from command line?

### ISSUE THREAD

**Issue by johnhopson** [SQLiteFlow/SQLiteFlow-Issues#252]
Is there a way to open a SQLite file from the command line?  e.g. open /Applications/SQLiteFlow.app "default.store"

 - Platform: macOS 15.1
 - SQLiteFlow: 6.5.1

**Comment by SQLiteFlow** [comment]
I believe it can already do this.
What problem are you met when you tried to do this?

**Comment by johnhopson** [comment]
Ahh.  You're right.  I got the Terminal command line wrong.  Below works.  Thanks for following up.

`open -a SQLiteFlow.app  "/Users/<name>/Library/Containers/<appname>/Data/Library/Application Support/default.store"`

**Comment by SQLiteFlow** [comment]
You're welcome and congrats!
And I'm curious why are you trying to open SQLiteFlow from Terminal, is this part of some kind of automation steps?

**Comment by johnhopson** [comment]
Yes.  I'm working on a SwiftData project and want visibility into the SQLite content for debugging.  The project logs this line for easy access.  Is there a better way to integrate with Xcode?  (I'm new to SwiftUI/SwiftData)

**Comment by SQLiteFlow** [comment]
> Is there a better way to integrate with Xcode?

That's depend on what you want to do eventually.

Assuming you're going to open a SQLite database on iOS simulator, I believe the Access Simulator feature will help, with this feature you don't have to log a line to know the database path. 

Access Simulator can let you open your App's Documents folder in Finder directly, then you can open your SQLite database by double clicking it, once it opened, I believe you don't have to open it again and again.

Screenshots attached.

<img width="602" alt="1" src="https://github.com/user-attachments/assets/833e1654-dd68-44d0-952f-d905a4333674">
<img width="453" alt="2" src="https://github.com/user-attachments/assets/40c34227-1ed3-4b89-a850-ec0dce35fac6">

**Comment by johnhopson** [comment]
Thanks for the heads up.  This project is actually a macOS project.  But that access tool will be helpful when I get to iOS development.

**Comment by SQLiteFlow** [comment]
Oh I see. If a project is a macOS project, then its document folder is fixed, so why not add the document folder to Finder's sidebar (this can be achieved in Finder by dragging the document folder and dropping it to the sidebar), after that, you can navigate to your database whenever you want conveniently. But this is a personal preference though.
